tests/integration/test_camera_picamera2.py

```diff
@@ -17,7 +16,20 @@ def main() -> int:
     """
     Main function.
     """
-    result, device = camera_factory.create_camera(camera_factory.CameraOption.PICAM2, 640, 480)
+
+    config = camera_config.PiCameraConfig(exposure_time=250, contrast=1.0, analogue_gain=64.0)
+    try:
+        assert config.exposure_time == 250
+        assert config.contrast == 1.0
+        assert config.analogue_gain == 64.0
+        assert config.lens_position is None
+    except AssertionError as e:
+        print(f"Configuration test failed: {e}")
+        return -1
```

Comment:
Same here, just keep the assert lines, no need to catch the exception since this is a test